mac_client.md

```diff
@@ -0,0 +1,93 @@
+# Podman Mac CLient tutorial
+
+## What is the Podman Mac Client
+
+First and foremost, the Mac Client is under heavy development. We are working on getting the
+Mac client to be packaged and run for a native-like experience. This is the setup tutorial
+for the Mac client at its current stage of development and packaging.
+
+The purpose of the Podman for Mac is to allow users to run Podman on a Mac. Since Podman is a Linux
+container runtime, The Mac client is actually a version of the [remote-client](remote_client.md),
+edited to that the client side works on a Mac machine, and connects to a Podman "backend" on a Linux
+machine, virtual or physical. The goal is to have a naitive-like experience when working with the Mac
+client, so the command line interface of the remote client is exactly thesame as the regular Podman
+commands with the exception of some flags being removed as they do not apply to the Mac client.
+
+## What you need
+To use the Mac client, you will need a binary built for MacOS and a Podman "backend" on a Linux machine;
```

Comment:
is there a minimum version of MacOS?  I think not, but thought I'd ask just in case.